cmd/submit/rbac.go

```diff
@@ -77,9 +77,9 @@ func getKubernetesApi() *k8sinterface.KubernetesApi {
 	}
 	return k8sinterface.NewKubernetesApi()
 }
-func getTenantConfig(credentials *cautils.Credentials, clusterName string, k8s *k8sinterface.KubernetesApi) cautils.ITenantConfig {
+func getTenantConfig(credentials *cautils.Credentials, clusterName string, customClusterName string, k8s *k8sinterface.KubernetesApi) cautils.ITenantConfig {
```

Comment:
Why do you pass both names? I will expect the `clusterName` to be ovverriden by `customeClusterName` before calling the function

Reply:
If `customClusterName` is not provided, then it sets the name as the original cluster's name (`clusterName`), instead of a custom name.
Else if a `customClusterName` is provided, then it overrides the `clusterName`